src/main/kotlin/no/nav/faktureringskomponenten/domain/models/EksternFakturaStatus.kt

```diff
@@ -36,5 +33,9 @@ class FakturaMottatt(
 
     @Column(name = "sendt", nullable = true)
     var sendt: Boolean? = false,
+
+    @ManyToOne(fetch = FetchType.LAZY, optional = false)
+    @JoinColumn(name = "faktura_id", nullable = false)
+    var faktura: Faktura? = null
```

Comment:
Ser ikke ut som vi kommer til å prøve å få tak i en faktura fra en status. Tror ikke at vi trenger denne mappingen.

Reply:
Jeg og Isa sleit litt med testene om det ikke var det. Da fikk vi feil som sa at vi manglet faktura_id

Reply:
Det virker for meg som man kunne bare lagre ekstern faktura status fra faktura alene uten å ha denne.
Ikke noe blokkerende, men vi bør vel prøve å unngå unødvendig mapping